cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -179,6 +179,10 @@
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.10</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
```

Comment:
I remember ordering of dependency matter in certain maven version. Probably should add this dependency before Kafka just to be safe?